ocs_ci/ocs/hsbench.py

```diff
@@ -357,6 +358,7 @@ def validate_reshard_process(self):
             CommandFailed: If reshard process fails
 
         """
+        bucket_name = self.bucket_prefix + "000000000000"
```

Comment:
Why should we add ` "000000000000"`?

Reply:
This is default naming convention which created by hsbench with user defined prefix.  In this test case, prefix is set as "bp01" and "000000000000" will be the first bucket created.

Reply:
This test case is for a single bucket creation, will address this as enhancement in separate a PR. 
Enhance PR is tracked here [PR # 6947](https://github.com/red-hat-storage/ocs-ci/issues/6974)